es/tests/test_dbapi.py

```diff
@@ -102,6 +101,9 @@ def test_execute_select_all(self):
         DBAPI: Test execute select all (*)
         """
         rows = self.cursor.execute("select * from flights LIMIT 10").fetchall()
+        # Make sure we have a list of tuples
+        self.assertEqual(type(rows), type(list()))
+        self.assertEqual(type(rows[0]), type(tuple()))
```

Comment:
I tried googling if there's a more pythonic way of doing this, but wasn't able to come up with anything better than this approach.